app-admin/yadm/yadm-3.1.1.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..9} )
+inherit bash-completion-r1 python-any-r1
+
+DESCRIPTION="A dotfile manager for the config files in your home folder"
```

Comment:
it's a folder on windows, in *nix it's a directory

Reply:
Will update with wording from the upstream.